mars/services/task/execution/ray/executor.py

```diff
@@ -220,6 +244,10 @@ async def execute_subtask_graph(
         tile_context: TileContext,
         context: Any = None,
     ) -> Dict[Chunk, ExecutionChunkResult]:
+        if self._cancelled is True:  # pragma: no cover
+            raise asyncio.CancelledError()
+        self._execute_subtask_graph_task = asyncio.current_task()
```

Comment:
```suggestion
        self._execute_subtask_graph_aiotask = asyncio.current_task()
```
The aio task, mars task and ray task start to get confusing.  Better to use a more qualified name

Reply:
Good suggestion.